runner/ollamarunner/runner.go

```diff
@@ -814,6 +822,15 @@ func (s *Server) loadModel(
 		panic(err)
 	}
 
+	// TODO(jmorganca): free this when no longer used
+	// TODO(jmorganca): remove this once llama.cpp's
+	// grammar parsing is removed
+	vocab, err := llama.LoadVocabFromFile(mpath)
```

Comment:
This looks like it could effect cold start time, if it seems like a noticeable impact we could lazy load this on the first grammar request.